scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -55,6 +55,18 @@
   }
 }
 
+.nav-pills {
+  margin-bottom: 15px;
```

Comment:
that's the design of a specific page, not a design of the theme in general. Bootstrap doesn't make any assumption on what margin nav-pills should have, so shoreditch shouldn't either. Do not add exception unless they are absolutely necessary